lib/DataIntegrityProof.js

```diff
@@ -304,7 +304,12 @@ export class DataIntegrityProof extends LinkedDataProof {
       }
       if(proof.created !== undefined) {
         if(!util.isW3cDate(proof.created)) {
-          throw new Error('Invalid XML TimeStamp');
+          throw new Error('Invalid XML TimeStamp for "proof.created"');
```

Comment:
We should include the value here in the error message:

```suggestion
          throw new Error(
            `"proof.created" ("${proof.created}") ` +
            'must be an XSD dateTimeStamp.');
```